spec/rails_helper.rb

```diff
@@ -1,12 +1,12 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
-require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
+require 'spec_helper'
```

Comment:
Why move this down three lines?

Reply:
Was advised to do this in the email_spec gem instructions... 
![image](https://user-images.githubusercontent.com/1826464/38176538-c5e47bc2-35e8-11e8-9d22-694b9dbca74a.png)